src/internal/SetBuildNumber/Directory.Packages.props.pp

```diff
@@ -53,7 +53,7 @@
     <PackageVersion Include="System.Reflection.Metadata" Version="1.8.1" />
     <PackageVersion Include="System.Security.Principal.Windows" Version="4.7.0" />
     <PackageVersion Include="System.Text.Encoding.CodePages" Version="4.7.1" />
-    <PackageVersion Include="System.Text.Json" Version="6.0.9" />
+    <PackageVersion Include="System.Text.Json" Version="6.0.10" />
```

Comment:
Would be good to put this in a separate commit.

Reply:
Done.